cmd/savepcdfiles/main.go

```diff
@@ -91,9 +102,13 @@ func savePCDFiles(ctx context.Context, port int, lidarComponent config.Component
 		return errors.New("no rplidar found with name: " + name)
 	}
 
+	if !utils.SelectContextOrWait(ctx, time.Second) {
+		return multierr.Combine(ctx.Err(), myRobot.Close(ctx))
+	}
+
 	// Run loop
 	for {
-		if !utils.SelectContextOrWait(ctx, time.Second) {
+		if !utils.SelectContextOrWait(ctx, time.Duration(timeDelta)*time.Millisecond) {
```

Comment:
Updated time delta to allow user to change/specify time to wait between measurments

Reply:
This is great!